internal/signing/azure_keyvault.go

```diff
@@ -24,16 +24,21 @@ import (
 	"fmt"
 	"io"
 	"math/big"
+	"net/url"
+	"os"
 	"strings"
 
 	"github.com/Azure/azure-sdk-for-go/services/keyvault/v7.0/keyvault"
-	"github.com/Azure/go-autorest/autorest/azure/auth"
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/go-autorest/autorest/adal"
+	"github.com/Azure/go-autorest/autorest/azure"
 	"github.com/google/exposure-notifications-server/internal/base64util"
 )
 
 // Compile-time check to verify implements interface.
 var _ KeyManager = (*AzureKeyVault)(nil)
 var _ crypto.Signer = (*AzureKeyVaultSigner)(nil)
+var keyvaultAuthorizer autorest.Authorizer
```

Comment:
if this should indeed only be initialized once, should it be protected by a mutex?

Reply:
It ended up being unnecessary so I took it out.

Reply:
Sorry, I see what you mean.  I think that might make sense.

Reply:
wrapping it up in the struct is likely better.

in practice, it will be a singleton

Reply:
I think I understood, does it look closer to what you had in mind now?

Reply:
this is fine - technically in the new form, doesn't need to be rolled into a struct, but let's go w/ this for now.